sdk/purview/Azure.Analytics.Purview.Sharing/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2023-06-30)
 
 ### Features Added
 
-### Breaking Changes
```

Comment:
nit: there are a couple breaking changes in this new beta that aren't mentioned.

- should add the removal of the `skipToken` param from quite a few operations and a brief mention of what a customer can / should do to mitigate.
- should add a mention that the constructor now takes a Uri instead of a string

Reply:
Added some extra information, thanks for the comment Michael!